k8s/sonarqube/Makefile

```diff
@@ -0,0 +1,109 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+
+TAG ?= 7.6
+METRICS_EXPORTER_TAG ?= v0.5.1
+SRCTAG ?= 9.6-kubernetes
```

Comment:
`SRCTAG` -> `PROMETHEUS_SRCTAG` and add SonarQube specific tag SRCTAG

Reply:
updated tot POSTGRESQL_SRCTAG because I want specific version of postgreSQL here